### ISSUE TITLE
show causes incorrect parsing of command parameters

### ISSUE THREAD

**Issue by heksesang** [com-lihaoyi/mill#1648]
When using the `mill show` command with a `T.command`, it causes mill to incorrectly parse the command line parameters. The issue seems to only affect mill 0.9.10 and later.

It can be reproduced by creating the file `build.sc` with the following content:
```scala
import mill._, scalalib._

def release(repo: String) =
  T.command {
    println("test")
  }
```

And then running this on the command line: `mill -i show release --repo heksesang/mill`

The result will be:
```bash
[1/1] show 
1 targets failed
show Cannot resolve external module heksesang
```

**Comment by lefou** [comment]
Thanks for reporting. I've prepared a fix in #1649.

**Comment by lefou** [comment]
Closed by #1649
